fixbackend/auth/oauth_router.py

```diff
@@ -62,7 +62,7 @@ def generate_state_token(data: Dict[str, str], secret: SecretType, lifetime_seco
 # forked version of fastapi_users.router.oauth.get_oauth_router
 # to allow for redirect_url to be set via the JWT token
 def get_oauth_router(
-    oauth_client: BaseOAuth2[Any],
+    oauth_client: BaseOAuth2[Any] | GithubOauthClient,
```

Comment:
GithubOauthClient is an BaseOauth2? 
Since BaseOAuth2 can be a lot of other types,  it is rather confusing to define a union here.
Suggestion: remove the `| GithubOauthClient`